

drivers/scsi/eata_pio.c: In function `eata_pio_host_reset':
drivers/scsi/eata_pio.c:514: warning: implicit declaration of function `msleep'


Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/scsi/eata_pio.c |    2 ++
 1 files changed, 2 insertions(+)

diff -puN drivers/scsi/eata_pio.c~eata_pio-warning-fix drivers/scsi/eata_pio.c
--- 25/drivers/scsi/eata_pio.c~eata_pio-warning-fix	2004-08-04 19:31:16.478435744 -0700
+++ 25-akpm/drivers/scsi/eata_pio.c	2004-08-04 19:31:41.882573728 -0700
@@ -59,6 +59,8 @@
 #include <linux/interrupt.h>
 #include <linux/blkdev.h>
 #include <linux/spinlock.h>
+#include <linux/delay.h>
+
 #include <asm/io.h>
 
 #include <scsi/scsi.h>
_
